src/components/elements/typography.ts

```diff
@@ -92,6 +92,11 @@ export const BaseElement = css<BaseElementProps>`
     text-align: ${(props) => props.align || 'start'};
     padding: ${(props) => props.padding || ''};
     color: ${({ color }) => (color ? `var(--color-${color})` : 'var(--color-black-3)')};
+
+    &[data-color='red'] {
+        color: var(--color-red);
```

Comment:
why cant we use the color prop that used in line 94?

Reply:
Because of w3c is complaining about the `color` attribute , we are avoid using that.

Reply:
so in that case whats the point of line 94?
i mean i'm thinking that styled components doesn't add all of the attrs on the element unless we add them manually, is that right?

Reply:
@ako Yes, That is right. 
In this particular PR we are reducing the number of w3c errors on `Attribute color not allowed on element [p](https://html.spec.whatwg.org/multipage/#the-p-element) at this point.`  
and line 94 still valid but with the cost of w3c errors. if we want to remove the all errors from w3c on this we have to get rid of the `color` attribute from <Header> component and use data- attributes instead.